src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { setQuery, setStatus } from '../../features/filter';
 
 export const TodoFilter: React.FC = () => {
+  // const [queryString, setQueryString] = useState('');
```

Comment:
The commented-out `useState` hook for `queryString` is not used in the current implementation. If it's not needed, consider removing it to keep the code clean.